docs/sources/operators-guide/mimir-runbooks/_index.md

```diff
@@ -73,29 +73,19 @@ How to **fix** it:
 - Run the following **instant query** to find tenants that might cause higher pressure on some ingesters:
 
   ```
-  (
-    sum by(user) (cortex_ingester_memory_series_created_total{namespace="<namespace>"}
-    -
-    cortex_ingester_memory_series_removed_total{namespace="<namespace>"})
+  topk by (pod) (
+    5,
+    (
+        sum by (user, pod) (
+            cortex_ingester_memory_series_created_total{namespace="<namespace>"} - cortex_ingester_memory_series_removed_total{namespace="<namespace>"}
+        )
+    ) and on (pod) (
+        topk(
+            3,
+            sum by (pod) (cortex_ingester_memory_series_created_total{namespace="<namespace>"} - cortex_ingester_memory_series_removed_total{namespace="<namespace>"})
+        )
+    )
```

Comment:
is this change supposed to be part of this PR?

Reply:
haha, it's not, I also `git reset --hard`, but I forgot to force push it back